packages/scripts/utils/config.js

```diff
@@ -17,7 +17,21 @@ const hasJestConfig = () =>
 	hasProjectFile( 'jest.config.json' ) ||
 	hasPackageProp( 'jest' );
 
+const hasWebpackConfig = () => hasCliArg( '--config' ) ||
+	hasProjectFile( 'webpack.config.js' ) ||
+	hasProjectFile( 'webpack.config.babel.js' );
+
+const getWebpackArgs = ( additionalArgs = [] ) => {
+	const webpackArgs = [ ...getCliArgs() ];
```

Comment:
Nit: it's perfectly fine to avoid using array spread operator:
```suggestion
	const webpackArgs = getCliArgs();
```